### PR TITLE
9 new icons

### PR DESCRIPTION
**Pull request by RaduBratan**
# Description
More icons for apps that I've got on my phone.

### Icons added
* 24pay (`com.twentyfourpay`)
* Android 12 Widget Pack (`android12widget.kustom.pack`)
* AntenaPLAY (`ro.antenaplay.app`)
* F1 Mobile Racing (`com.codemasters.F1Mobile`)
* Info TB (`ro.stbsa`)
* MusicCast (`com.yamaha.av.musiccastcontroller`)
* MyEdenred (`com.edenred.eq.myedenred`)
* Stay Fit Gym (`ro.stayfit`)
* Youth TB (`ro.stbsa.ticketing`)

### CODE REVIEW

File: svgs/twentyfourpay.svg
Comment (RaduBratan) [inline svgs/twentyfourpay.svg:1]: I should move the "y" 1-2 px to the right.

File: svgs/youth_tb.svg
Comment (RaduBratan) [inline svgs/youth_tb.svg:1]: The "th" is practically invisible. I should increase the distance between the U's lines and "th", while also making the latter bigger.

File: svgs/stay_fit_gym.svg
Comment (RaduBratan) [inline svgs/stay_fit_gym.svg:1]: The gaps appear thinner than the outlines. I have to increase every element's width.

File: svgs/youth_tb.svg
Comment (RaduBratan) [inline svgs/youth_tb.svg:1]: This icon contains too many segments and stroke widths. I should simplify it.

File: svgs/f1_mobile_racing.svg
Comment (RaduBratan) [inline svgs/f1_mobile_racing.svg:1]: "F1" should be way bigger than "MR". Also, the "R" should be a bit wider.

File: svgs/android_12_widget_pack.svg
Comment (x9136) [inline svgs/android_12_widget_pack.svg:1]: Here you could avoid the fill without loss of quality (and even with an improvement) by enlarging the android a bit and making the eyes 6×6px with a 6px stroke.

File: svgs/antenaplay.svg
Comment (x9136) [inline svgs/antenaplay.svg:1]: The original compensated for the void on the right by shifting the icon there a bit. I propose to move the icon a little to the right here too.

**Pull request merged**